src/lib/tls/tls_extensions.h

```diff
@@ -3,49 +3,56 @@
 * (C) 2011,2012,2016,2018,2019 Jack Lloyd
 * (C) 2016 Juraj Somorovsky
 * (C) 2016 Matthias Gierlings
+* (C) 2021 Elektrobit Automotive GmbH
+* (C) 2022 René Meusel, Hannes Rantzsch - neXenio GmbH
 *
 * Botan is released under the Simplified BSD License (see license.txt)
 */
 
 #ifndef BOTAN_TLS_EXTENSIONS_H_
 #define BOTAN_TLS_EXTENSIONS_H_
 
+#include <botan/ocsp.h>
```

Comment:
Is this new include necessary here?

Reply:
No, neither for TLS 1.3. That must be an artefact of some intermediate spike. Good catch!